docker/localsettings.py

```diff
@@ -102,8 +102,8 @@
         # for production this ought to be set to true on your configured couch instance
         'COUCH_HTTPS': False,
         'COUCH_SERVER_ROOT': 'couch:5984',  # 6984 for https couch
-        'COUCH_USERNAME': '',
-        'COUCH_PASSWORD': '',
+        'COUCH_USERNAME': 'commcarehq',
+        'COUCH_PASSWORD': 'commcarehq',
```

Comment:
@emord 
In docker/localsettings.py I saw that we had previously set the couch username and password and then you reverted it.  What was causing the failures that led you to revert it?

Reply:
I think there were some issues with the initial rollout of these passwords and I reverted it as a short term measure. 

Looks like it might be caused because of a difference with how travis is instantiated

https://github.com/dimagi/commcare-hq/blob/master/docker/hq-compose-services.yml#L12 uses password

https://github.com/dimagi/commcare-hq/blob/master/docker/hq-compose.yml#L39 uses admin party. 

may need to change the second one to use a password